package.json

```diff
@@ -50,7 +50,7 @@
     "prettier:fix": "prettier --write \"./{libs,apps}/**/*.{ts,json,md,scss,html}\"",
     "e2e": "ng e2e",
     "dist": "npm run dist:cookbook",
-    "dist:cookbook": "ng build cookbook --prod --aot --progress false",
+    "dist:cookbook": "ng build cookbook --configuration production",
```

Comment:
For some reason this is appearing in the diff - but it is already present on target branch